### ISSUE TITLE
Sound volume locked to 0

### ISSUE THREAD

**Issue by eleloi** [michelelacorte/FlickLauncher#32]
My phone has the volume locked to 0. When I try to up with the volume up button the system show the volume bar, but nothing happens.

I can move the slider with the  finger, but the volume don't work, and when i press the VolumeUp button the slider goes back to 0.

My phone is Ok, I uninstalled the Flick launcher and the volume just worked again.

**Comment by eleloi** [comment]
I've just read that:

Michele Lacorte
Propietario
+1
This bug is happen because ok google voice recognition causes conflict with audio manager of phone, just disable it! I have to fix this

I think it's the same Issue.

**Comment by michelelacorte** [comment]
Solved?

**Comment by eleloi** [comment]
No, keeps failing,with ok Google recognition off it works, but with on option it fails.

**Comment by eleloi** [comment]
I'm with 159 build

**Comment by michelelacorte** [comment]
Yes It's know bug, I will fix soon!
